src/PactNet.Abstractions/Matchers/ArrayContainsMatcher.cs

```diff
@@ -0,0 +1,17 @@
+using System.Text.Json.Serialization;
+
+namespace PactNet.Matchers;
+
+public class ArrayContainsMatcher : IMatcher
```

Comment:
issue: The public API should be documented

Reply:
Noted, will amend